<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSettings</class>
 <widget class="QDialog" name="ServerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icon/app.ico</normaloff>:/icon/app.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>This Server</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="sv_address">
            <property name="text">
             <string notr="true">127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sv_port">
            <property name="suffix">
             <string notr="true"/>
            </property>
            <property name="prefix">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>7777</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Server DB</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Address: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>DB user:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>DB password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>DB name: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ODBC Driver: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>DB Charset: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="db_address">
            <property name="text">
             <string notr="true">127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="db_port">
            <property name="suffix">
             <string notr="true"/>
            </property>
            <property name="prefix">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>5432</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="db_user">
            <property name="text">
             <string notr="true">qt_chat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="db_pass">
            <property name="text">
             <string notr="true">qt_chat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="db_name">
            <property name="text">
             <string notr="true">qt_chat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="db_odbc_driver">
            <property name="text">
             <string notr="true">PostgreSQL Unicode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="db_charset">
            <property name="text">
             <string notr="true">utf8</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>To apply the settings, you need to restart the server</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
